### PR TITLE
Chore/add revoke description to docs

### PR DESCRIPTION
**Pull request by george42-ctds**
### Improvements

* Add a description of how to remove access in the `authorization` document.

### CODE REVIEW

**Review by smvgarcia — APPROVED** [review]
Love it! thanks, George!

File: docs/authorization.md [hunk 1]
```diff
@@ -95,3 +95,22 @@ authz:
...
+The body of the request should have the `username` and `policy_id`, for example
+
+```json
+{
+    "username": "johndoe@example.com",
+    "policy_id": "dataset_D_reader",
+}
+```
+
+The user's access will be revoked when the new request has been approved by an administrator.
```
Comment (paulineribeyre) [inline docs/authorization.md:116]:
```suggestion
Just like access requests, revocation requests must be approved before they take effect. The user's access will be revoked when the new request has been approved by an administrator.

**IMPORTANT NOTE:** Requestor can only revoke access that has been granted through Requestor.
What does this mean? Access granted through the [user.yaml file](https://github.com/uc-cdis/fence/blob/master/docs/user.yaml_guide.md) cannot be revoked by Requestor. Similarly, removing access through the user.yaml file will not revoke access granted by Requestor.
```

**Review by paulineribeyre — CHANGES_REQUESTED** [review]

**Review by paulineribeyre — DISMISSED** [review]
Thanks!

File: docs/authorization.md (hunk 1 above)
Comment (george42-ctds) [reply docs/authorization.md]: Nice! Thank you for the clarifications!

**Review by paulineribeyre — APPROVED** [review]

**Comment by george42-ctds** [comment]
The poetry.lock was reverted to master because the dependency upgrades were handled in the dependabot PRs.

**Pull request merged**